openapi.yml

```diff
@@ -435,6 +436,7 @@ components:
               description: Status of the descriptive element value relative to other instances
                 of the element.
               type: string
+              # type: integer
```

Comment:
Cruft?

Reply:
See #154 . It's in the conceptual model but can't yet be implemented due to a bug elsewhere.

Reply:
Can you add that to the comment?